<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
	<class>MainWindow</class>
	<widget class="QMainWindow" name="MainWindow">
		<property name="geometry">
			<rect>
				<x>0</x>
				<y>0</y>
				<width>1000</width>
				<height>850</height>
			</rect>
		</property>
		<property name="windowTitle">
			<string>MainWindow</string>
		</property>
		<property name="styleSheet">
			<string notr="true">background-color: #ffffff</string>
		</property>
		<widget class="QWidget" name="centralwidget">
			<widget class="QPlainTextEdit" name="outputTB">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>340</y>
						<width>870</width>
						<height>200</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPlainTextEdit{
	border: 3px solid #ffac74;
	border-radius: 10px; 
	color: #ac6e23;
	background-color: #fff7ea;
	padding: 10px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}
</string>
				</property>
				<property name="plainText">
					<string/>
				</property>
			</widget>
			<widget class="QLabel" name="label_3">
				<property name="geometry">
					<rect>
						<x>540</x>
						<y>110</y>
						<width>251</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">color: #ac6e23;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;</string>
				</property>
				<property name="text">
					<string>Key</string>
				</property>
			</widget>
			<widget class="QWidget" name="widget" native="true">
				<property name="geometry">
					<rect>
						<x>0</x>
						<y>0</y>
						<width>1000</width>
						<height>80</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QWidget{
	background-color: #ffdab5;
}
</string>
				</property>
				<widget class="QLabel" name="label">
					<property name="geometry">
						<rect>
							<x>220</x>
							<y>20</y>
							<width>600</width>
							<height>41</height>
						</rect>
					</property>
					<property name="styleSheet">
						<string notr="true">color: #ac6e23;
font: 500 18pt &quot;MS Shell Dlg 2&quot;;</string>
					</property>
					<property name="text">
						<string>Tugas 1 Kriptokoding: Cipher Machine</string>
					</property>
					<property name="alignment">
						<set>Qt::AlignCenter</set>
					</property>
				</widget>
			</widget>
			<widget class="QPlainTextEdit" name="textInput">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>150</y>
						<width>410</width>
						<height>120</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPlainTextEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid #ffcca0;
	color: #ac6e23;
	background-color: #fff7ea;
	padding: 10px;
}

QPlainTextEdit:hover {
	border: 3px solid #ffac74;
}
</string>
				</property>
				<property name="plainText">
					<string/>
				</property>
			</widget>
			<widget class="QPushButton" name="inputBut">
				<property name="geometry">
					<rect>
                         <!-- Input your file here button -->
						<x>70</x>
						<y>610</y>
						<width>410</width>
						<height>120</height>
					</rect>
				</property>
				<property name="autoFillBackground">
					<bool>false</bool>
				</property>
				<property name="styleSheet">
					<string notr="true">QPushButton {
	font: 500 17pt &quot;MS Shell Dlg 2&quot;;
	border: #ac6e23;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffdab5;
	color: #ac6e23;
}

QPushButton:hover {
	background-color: #ffac74;
}

QPushButton:pressed {
	background-color: #e69b69;
	color: #fff;
}
</string>
				</property>
				<property name="text">
					<string>Atau, masukkan file ke sini!</string>
				</property>
			</widget>
			<widget class="QComboBox" name="cipheroption">
				<property name="geometry">
					<rect>
						<x>80</x>
						<y>565</y>
						<width>391</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QComboBox{
	background-color: #ebebeb;
	font: 75 10pt &quot;MS Shell Dlg 2&quot;;
	padding-left: 10px;
	border: none;
}
</string>
				</property>
				<item>
					<property name="text">
						<string>Vigenere Cipher Encrypt</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>Vigenere Cipher Decrypt</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>Extended Vigenere Cipher Encrypt</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>Extended Vigenere Cipher Decrypt</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>Playfair Cipher Encrypt</string>
					</property>
				</item>
				<item>
					<property name="text">
						<string>Playfair Cipher Decrypt</string>
					</property>
				</item>
			</widget>
			<widget class="QLabel" name="label_2">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>110</y>
						<width>131</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">color: #ac6e23;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;</string>
				</property>
				<property name="text">
					<string>Plain Text</string>
				</property>
			</widget>
			<widget class="QPushButton" name="cryptBut"> 
				<property name="geometry">
					<rect>
                         <!-- Run Button -->
						<x>530</x>
						<y>610</y>
						<width>410</width>
						<height>120</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPushButton {
	font: 700 20pt &quot;MS Shell Dlg 2&quot;;
	border: none;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffdab5;
	color: #ac6e23;
}

QPushButton:hover {
	background-color: #ffac74;
}

QPushButton:pressed {
	background-color: #e69b69;
	color: #fff;
}
</string>
				</property>
				<property name="text">
					<string>Run!</string>
				</property>
			</widget>
			<widget class="QLabel" name="label_4">
				<property name="geometry">
					<rect>
						<x>70</x>
						<y>300</y>
						<width>300</width>
						<height>31</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">color: #ac6e23;
font: 550 14pt &quot;MS Shell Dlg 2&quot;;</string>
				</property>
				<property name="text">
					<string>Hasil Cipher Text</string>
				</property>
			</widget>
			<widget class="QPlainTextEdit" name="keyInput">
				<property name="geometry">
					<rect>
						<x>530</x>
						<y>150</y>
						<width>410</width>
						<height>120</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPlainTextEdit{
	font: 10pt &quot;MS Shell Dlg 2&quot;;
	border: 1px solid #ffcca0;
	color: #ac6e23;
	background-color: #fff7ea;
	padding: 10px;
}

QPlainTextEdit:hover {
	border: 3px solid #ffac74;
}
</string>
				</property>
				<property name="plainText">
					<string/>
				</property>
			</widget>
			<widget class="QPushButton" name="enigmaBut">
				<property name="geometry">
					<rect>
						<x>530</x>
						<y>750</y>
						<width>190</width>
						<height>65</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPushButton {
	font: 700 11pt &quot;MS Shell Dlg 2&quot;;
	border: none;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffffff;
	border: 3px solid #ffac74;
	color: #ac6e23;
}

QPushButton:hover {
	background-color: #ffdab5;
}

QPushButton:pressed {
	background-color: #e69b69;
	color: #fff;
}
</string>
				</property>
				<property name="text">
					<string>Enigma Machine</string>
				</property>
			</widget>
			<widget class="QPushButton" name="otpBut">
				<property name="geometry">
					<rect>
						<x>750</x>
						<y>750</y>
						<width>190</width>
						<height>65</height>
					</rect>
				</property>
				<property name="styleSheet">
					<string notr="true">QPushButton {
	font: 700 11pt &quot;MS Shell Dlg 2&quot;;
	border: none;
	border-left 2px solid transparent;
	border-radius: 15px; 
	background-color: #ffffff;
	border: 3px solid #ffac74;
	color: #ac6e23;
}

QPushButton:hover {
	background-color: #ffdab5;
}

QPushButton:pressed {
	background-color: #e69b69;
	color: #fff;
}
</string>
				</property>
				<property name="text">
					<string>One Time Pad</string>
				</property>
			</widget>
		</widget>
		<widget class="QMenuBar" name="menubar">
			<property name="geometry">
				<rect>
					<x>0</x>
					<y>0</y>
					<width>1000</width>
					<height>26</height>
				</rect>
			</property>
		</widget>
		<widget class="QStatusBar" name="statusbar"/></widget>
	<resources/>
	<connections/>
</ui>
